### ISSUE TITLE
NTR: Pyruvate kinase activity high in erythrocytes (Lab: blood chemistry: enzymes)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8231]
New term request Pyruvate kinase activity high in erythrocytes (elthat_130818190042): RBCs, red blood cells; hyperactivity

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000561
- label: Elevated red cell pyruvate kinase activity
- parents: [Abnormal erythrocyte enzyme level (HP:0030272)](https://hpo.jax.org/app/browse/term/HP:0030272)
- pmid: 
- def: Activity or concentration of pyruvate kinase (EC 2.7.1.40) in erythrocytes above the upper limit of normal. Pyruvate kinase (EC 2.7.1.40) is a glycolytic enzyme that catalyzes the transphosphorylation from phosphoenolpyruvate (PEP) to ADP, yielding pyruvate and ATP.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8231
- synonyms:
